docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md

```diff
@@ -100,13 +102,20 @@ Defines one or more switches used by the <xref:System.AppContext> class to provi
+|`"Switch.System.Windows.Controls.Text.`<br/>`UseAdornerForTextboxSelectionRendering`|Determines whether non-Adorner based selection rendering is available for the <xref:System.Windows.Controls.TextBox> and <xref:System.Windows.Controls.PasswordBox> controls to prevent occluded text (`false`), or whether text is rendered only in the Adorner layer (`true`).|.NET Framework 4.7.2|
 |`Switch.System.Windows.DoNotScaleForDpiChanges`|Determines whether DPI changes occur on a per-system (a value of `false`) or per-monitor basis (a value of `true`).|.NET Framework 4.6.2|
+|`Switch.System.Windows.Forms.`<br/>`DomainUpDown.UseLegacyScrolling`|Determines whether the developer needs to specially handle the <xref:System.Windows.Forms.DomainUpDown.UpButton?displayProperty=nameWithType> action when control text is present. `true` to handle the <xref:System.Windows.Forms.DomainUpDown.UpButton> action; `false` for the <xref:System.Windows.Forms.DomainUpDown.UpButton?displayProperty=nameWithType> and <xref:System.Windows.Forms.DomainUpDown.DownButton?displayProperty=nameWithType> actions to be properly in sync.|.NET Framework 4.7.2|
 |`Switch.System.Windows.Forms.`<br />`DontSupportReentrantFilterMessage`|Opts out of the code that allows a custom <xref:System.Windows.Forms.IMessageFilter.PreFilterMessage%2A?displayProperty=nameWithType> implementation to safely filter messages without throwing an exception when the <xref:System.Windows.Forms.Application.FilterMessage%2A?displayProperty=nameWithType> method is called. For more information, see [Mitigation: Custom IMessageFilter.PreFilterMessage Implementations](~/docs/framework/migration-guide/mitigation-custom-imessagefilter-prefiltermessage-implementations.md).|.NET Framework 4.6.1|  
-|`Switch.System.Windows.Input.Stylus.`<br/>`EnablePointerSupport`|Determines whether an optional `WM_POINTER`-based touch/stylus stack is enabled in WPF applications. For more information, see [Mitigation: Pointer-based Touch and Stylus Support](Mitigation:%20Pointer-based%20Touch%20and%20Stylus%20Support.md) | 
+|`Switch.System.Windows.Forms.`<br/>`UseLegacyContextMenuStripSourceControlValue`||.NET Framework 4.7.2|Determines whether the <xref:System.Windows.Forms.ContextMenuStrip.SourceControl?displayProperty=nameWithType> property returns the source control when the user opens the menu from a nested <xref:System.Windows.Forms.ToolStripMenuItem> control. `true` to return `null`, the legacy behavior; `false` to return the source control.|.NET Framework 4.7.2|
+|`Switch.System.Windows.Input.Stylus.`<br/>`EnablePointerSupport`|Determines whether an optional `WM_POINTER`-based touch/stylus stack is enabled in WPF applications. For more information, see [Mitigation: Pointer-based Touch and Stylus Support](Mitigation:%20Pointer-based%20Touch%20and%20Stylus%20Support.md)|.NET Framework 4.7|
```

Comment:
not related to your PR: broken link for mitigation (interesting that it doesn't show up in the build report)